commit 9abafb2cc34ff1dc935de2b9e08cb707e4a660f7 (refs/changes/01/2801/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-10-17T15:24:16-07:00 (2 years ago)
    
    DOCKER-1118 do not set min_platform for docker and lx images

diff --git a/src/img/CHANGES.md b/src/img/CHANGES.md
index 6c8cea16..b1c26a89 100644
--- a/src/img/CHANGES.md
+++ b/src/img/CHANGES.md
@@ -6,6 +6,10 @@ Known issues:
   Docker Registry v1, which is now deprecated.
 
 
+## 3.7.4
+
+- DOCKER-1118 docker build should not set image min_platform
+
 ## 3.7.3
 
 - OS-6383 Fix a possible crash in 'imgadm import ...'.
diff --git a/src/img/lib/imgadm.js b/src/img/lib/imgadm.js
index 722d0ad5..c18f7ee8 100644
--- a/src/img/lib/imgadm.js
+++ b/src/img/lib/imgadm.js
@@ -3461,7 +3461,7 @@ IMGADM.prototype.createImage = function createImage(options, callback) {
                     }
                 });
             }
-            if (vmInfo.brand !== 'kvm' /* i.e. this is a smartos image */
+            if ((vmInfo.brand === 'joyent' || vmInfo.brand === 'joyent-minimal')
                 && !(options.manifest.requirements
                     && options.manifest.requirements.min_platform))
             {
diff --git a/src/img/package.json b/src/img/package.json
index 02fabf3a..6c3234a5 100644
--- a/src/img/package.json
+++ b/src/img/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgadm",
   "description": "Manage SmartOS virtual machine images.",
-  "version": "3.7.3",
+  "version": "3.7.4",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
